spec/models/neighborhood_spec.rb

```diff
@@ -47,4 +47,25 @@
       expect(neighborhood.time_estimates.first.product_type).to eq("uberXL")
     end
   end
+
+  context '#current_time_estimate' do
+    it "returns the most recent time estimate based on created_at" do
+      uberx = "uberx"
+      older = neighborhood.time_estimates.create(product_type: uberx,
+                                         time: 3000,
+                                         created_at: Date.today - 1.day)
+      newer = neighborhood.time_estimates.create(product_type: uberx,
+                                         time: 2000,
+                                         created_at: Date.today)
+      expect(neighborhood.current_time_estimate(uberx)).to eq(newer)
+    end
+
+    it 'return time estimates that match the type passed in' do
+      uberXL = neighborhood.time_estimates.create(product_type: "uberXL",
+                                         time: 3000)
+      uberX = neighborhood.time_estimates.create(product_type: "uberX",
```

Comment:
Useless assignment to variable - `uberX`. Did you mean `uberXL`?<br>Use snake_case for variable names.